tests/providers/databricks/operators/test_databricks_sql.py

```diff
@@ -268,16 +266,18 @@ def test_return_value_serialization():
         ),
     ],
 )
-def test_exec_write_file(return_last, split_statements, sql, descriptions, hook_results, do_xcom_push):
+def test_exec_write_file(
+    return_last, split_statements, sql, descriptions, hook_results, do_xcom_push, tmp_path
+):
     """
     Test the execute function in case where SQL query was successful and data is written as CSV
     """
     with patch("airflow.providers.databricks.operators.databricks_sql.DatabricksSqlHook") as db_mock_class:
-        tempfile_path = tempfile.mkstemp()[1]
+        path = tmp_path / "testfile"
         op = DatabricksSqlOperator(
             task_id=TASK_ID,
             sql=sql,
-            output_path=tempfile_path,
+            output_path=path,
```

Comment:
```suggestion
            output_path=os.fspath(path),
```

or alternatively DatabricksSqlOperator should be modified to support Path.

Reply:
Done. I'd tackle the code in a different PR.